tests/container_disk_test.go

```diff
@@ -220,4 +220,90 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			})
 		})
 	})
+
+	Describe("Simulate an upgrade from a version where ImageVolume was disabled to a version where it is enabled", Serial, func() {
+		BeforeEach(func() {
+			v, err := getKubernetesVersion()
+			Expect(err).ToNot(HaveOccurred())
+			if v < "1.33" {
+				// Skip the test if the Kubernetes version is lower than 1.33
+				// ImageVolume won't work for versions < 1.33 because of this bug:
+				// https://github.com/kubernetes/kubernetes/pull/130394
+				// Additionally there is currently no way to enable k8s ImageVolume FG
+				// through kubevirtci. It is enabled by default since 1.33.
+				Skip("this test requires Kubernetes version >= 1.33")
```

Comment:
Can we use some kind of decorator to avoid the skip?
More generally, where do we want these tests to run? Do we want an alpha feature test to gate presubmits?

Perhaps what we can do is introduce a "non-matured" decorator and run these tests only on periodic lanes?

cc @dhiller